src/app/data/appConfig.js

```diff
@@ -5,7 +5,8 @@ const appConfig = {
   appTitle: 'NYPL | Discovery',
   appName: 'discovery',
   displayTitle: process.env.DISPLAY_TITLE || 'Shared Collection Catalog',
-  baseUrl: '/research/collections/shared-collection-catalog',
+  baseUrl: process.env.BASE_URL || '/research/collections/shared-collection-catalog',
+  previousBaseUrl: process.env.PREVIOUS_BASE_URL,
```

Comment:
Maybe this could be static? I don't think it will need to anything other than '/research/collections/shared-collection-catalog' for awhile

Reply:
I like that better. Changed to static.

Given that our "previousBaseUrl" will also be out current "baseUrl", added check in server.js to ensure we only redirect when they're different.